### DISCUSSION TITLE
Add Linux Support [Complete]

### DISCUSSION THREAD

**Discussion by ivsbex** [Jenova-Framework/J.E.N.O.V.A#4] [Ideas]
Microsoft still tries to make Windows more unusable with each update and Linux is slowly gaining its popularity, especially among developers. So why not to add support for Linux and non-proprietary software like `vs-code` or `vim` instead of Visual Studio (while still probably leaving it as an option).

**Comment by TheAenema** [comment]
@ivsbex Hi, I am not a Linux developer, I made the project open-source for the reason, So let's hope someone migrate it for other platforms :)

**Comment by TheAenema** [comment]
I'm working on an abstraction layer and platform isolation for Jenova to facilitate migration to other platforms, including Linux. 
It's currently work in progress at [Jenova Runtime - Abstract API](https://github.com/Jenova-Framework/Jenova-Runtime/tree/abstract-api)

**Comment by TheAenema** [comment]
**Update :** Jenova is now compatible with Linux and gcc/clang compilers. While many components are still under development, they are slated for future updates.

**Comment by TheAenema** [comment]
**Update :** 90% of Jenova Source Code Ported to Linux

**Comment by TheAenema** [comment]
# Linux Port Reached 100% : Completed
After months of hard work Jenova Runtime is finally ported to Linux x64 Operating System completely!

## Jenova Linux comes with all core functions including :
- C++ Scripts/C++ Headers Support
- Nested Extension Development
- Sakura Hot-Reloading System
- Real-Time Hot-Reloading Support
- Both NitroJIT & Meteora Interpreter Backends are Available
- Property, Class Name and Built-in Script Support
- Visual Studio Code Integration + Exporter
- Debug Information Generation + Debug Support
- Standard C++ 20 Support
- External Editor Support (NeoVim etc.)
- And more!
